build.rs

```diff
@@ -0,0 +1,6 @@
+use std::env;
+
+fn main() {
+    println!("cargo:rustc-env=TARGET={}", env::var("TARGET").unwrap());
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
I think this line will break cross-compilation: if I first compile for e.g. linux-gnu and then linux-musl (or Mac or whatever), the TARGET variable will still be set to linux-gnu. Or am I missing something?

Reply:
Sounds like `println!("cargo:rerun-if-changed-env=TARGET");` should work, but I'm not sure if it's the case for the variables that Cargo sets internally

Reply:
Yep, I've tested and it works. I'll make a separate PR.